API/Services/Tasks/Scanner/Parser/Parser.cs

```diff
@@ -840,73 +793,26 @@ public static string ParseComicChapter(string filename)
 
     private static string RemoveEditionTagHolders(string title)
     {
-        foreach (var regex in CleanupRegex)
-        {
-            var matches = regex.Matches(title);
-            foreach (Match match in matches)
-            {
-                if (match.Success)
-                {
-                    title = title.Replace(match.Value, string.Empty).Trim();
-                }
-            }
-        }
+        title = CleanupRegex.Replace(title, string.Empty);
 
-        foreach (var regex in MangaEditionRegex)
-        {
-            var matches = regex.Matches(title);
-            foreach (Match match in matches)
-            {
-                if (match.Success)
-                {
-                    title = title.Replace(match.Value, string.Empty).Trim();
-                }
-            }
-        }
+        title = MangaEditionRegex.Replace(title, string.Empty);
 
         return title;
     }
 
     private static string RemoveMangaSpecialTags(string title)
```

Comment:
Those are now one-liners. I left them as their own methods just to make the PR easier to read but we could just add their code into CleanTitle instead and avoid the risk of function call overhead albeit the compiler should optimize those away.